data/human/free worlds 1 start.txt

```diff
@@ -1429,6 +1429,7 @@ mission "FW Pirates: Attack 2"
 		dialog `You've landed on <planet>, but there are still pirates defending the system. All pirates in the system must be eliminated before <planet> can be taken.`
 	on complete
 		event "fw suppressed Greenrock"
+		set "fw greenrock invader"
```

Comment:
Can you not just check "FW Pirates: Attack 2: done"?

Reply:
No, I didn't think of that.